build.sbt

```diff
@@ -40,6 +40,9 @@ lazy val freestyle = (crossProject in file("freestyle")).
     libraryDependencies ++= Seq(
       "org.typelevel" %%% "cats-free" % "0.9.0",
       "org.scala-lang" % "scala-reflect" % scalaVersion.value,
+      "io.monix" %%% "monix-eval" % "2.2.1" % "test",
+      "io.monix" %%% "monix-cats" % "2.2.1" % "test",
```

Comment:
We have that instance there because `Future` comes with the standard lib and while I agree with you that it doesn ot belong there, it's easier for users to keep a single dependency to run programs that can be potentially parallelizable out of the box. Since `Future` is the most abused `IO` like type in scala we expect most Freestyle usage happening around it.